package.json

```diff
@@ -30,6 +30,7 @@
     "@babel/core": "^7.7.0",
     "@theforeman/builder": "^12.0.1",
     "@theforeman/eslint-plugin-foreman": "^12.0.1",
+    "@theforeman/eslint-plugin-rules": "^12.0.2",
```

Comment:
please add that to `package-exclude.json` please :)

Reply:
added, thanks

Reply:
I had to submit https://github.com/theforeman/foreman/pull/9779 for this.

Reply:
Uhmm, what happened here? It seems to be added here https://github.com/theforeman/foreman/pull/9765/files#diff-97f0a353de600204e809ec124bbc205b69ef828bbad153523a81472cea107aa5R6